src/app/services/sound.service.ts

```diff
@@ -1,31 +1,25 @@
-import { Injectable } from '@angular/core';
+import {Injectable} from '@angular/core';
 
-@Injectable({
-  providedIn: 'root'
-})
-export class SoundService {
+const BASE_PATH = '../../assets/sounds';
 
-  private newClueAudio;
-  private askFirstAlertAudio;
-  private isMuted: boolean;
+export const enum Sound {
+  NEW_CLUE = 'new-clue-alert.wav',
+  PROPOSED_CLUE = 'ask-first-alert.wav',
+  WIN = 'win.mp3',
+  LOSE = 'lose.mp3',
+}
```

Comment:
Wanted to make it easy to add new sounds and then just call `soundService.play(Sound.SOME_SOUND)`